### ISSUE TITLE
Size of magnet

### ISSUE THREAD

**Issue by bmelnychuk** [premnirmal/Magnet#16]
Is it possible to change the size of custom view, looks like my width and height are ignored.

**Comment by premnirmal** [comment]
Hi @bmelnychuk . Are you calling `setIconView(..)` with a view or a layout resource?

**Comment by bmelnychuk** [comment]
Hi, I tried both. This is layout i want to have for example
```
<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="300dp"
    android:layout_height="300dp">

    <ImageView
        android:id="@+id/profileImage"
        android:src="@mipmap/ic_launcher"
        android:layout_width="match_parent"
        android:layout_height="match_parent" />

....

</FrameLayout>
```
but the size is always the same

**Comment by premnirmal** [comment]
I see. So the size is always the same because the layout params are not carried over from your xml view to the Window where the view is added. I will post a fix where you can manually set the width and height of the magnet. Thanks for pointing this out!

**Comment by premnirmal** [comment]
@bmelnychuk I attempted a fix in this PR https://github.com/premnirmal/Magnet/pull/17. Do you mind verifying the fix?

**Comment by premnirmal** [comment]
Fixed in version `1.1.6`
